cmd/ecsRightSizeCluster.go

```diff
@@ -38,7 +38,7 @@ This function may scale a cluster up or down depending on services.`,
 		initAwsSess()
 		err := lib.RightSizeAsgForEcsCluster(AwsSess, cluster, atLeastServiceDesiredCount)
 		if err != nil {
-			os.Exit(1)
+			log.Fatalln(err.Error())
```

Comment:
I don't think you need the `.Error()`. Can be changed in multiple places too.

```suggestion
			log.Fatalln(err)
```